packages/rehype-katex/package.json

```
@@ -33,6 +33,7 @@
     "index.js"
```

the types also need to be included in the package

```suggestion
    "index.js",
    "types/index.d.ts"
```